### PR TITLE
Add more German translations

### PR DESCRIPTION
**Pull request by Psychpsyo**
This adds about half of the missing German translation strings while also fixing the alphabetical ordering and a typo in some of the English locale files.

### CODE REVIEW

File: website/public/locales/de/common.json [hunk 1]
```diff
@@ -1,36 +1,61 @@
 {
   "about": "Über",
   "account_settings": "Konto",
-  "admin_dashboard": "Admin-Bereich",
+  "admin_dashboard": "Admin-Dashboard",
+  "back_to_dashboard": "Zum Dashboard zurückkehren",
+  "bug_description": "Wenn Sie Daten beitragen wollten aber hier gelandet bist, melde bitte einen Fehler.",
```
Comment (andreaskoepf) [inline website/public/locales/de/common.json:6]: mixture of formal (Sie) and informal style ...

File: website/public/locales/de/common.json [hunk 2]
```diff
@@ -1,36 +1,61 @@
...
+  "bug_button": "Fehler Melden",
```
Comment (andreaskoepf) [inline website/public/locales/de/common.json:7]: "melden" lowercase

File: website/public/locales/de/common.json [hunk 3]
```diff
@@ -1,36 +1,61 @@
...
   "messages": "Nachrichten",
+  "messages_dashboard": "Nachrichten-Dashboard",
   "more_information": "Weitere Informationen",
   "no": "Nein",
+  "output": "Ausgabe",
+  "parameters": "Parameter",
   "privacy_policy": "Datenschutz-Bestimmungen",
+  "prompt": "Prompt",
   "report_a_bug": "Einen Fehler melden",
+  "retry": "Erneut Versuchen",
```
Comment (andreaskoepf) [inline website/public/locales/de/common.json:38]: "versuchen" lowercase

File: website/public/locales/de/common.json [hunk 4]
```diff
@@ -1,36 +1,61 @@
...
+  "review": "Review",
+  "send": "Senden",
+  "sorry_404": "Sorry, die Seite nach der sie suchen existiert nicht.",
```
Comment (andreaskoepf) [inline website/public/locales/de/common.json:41]: "Sie" .. in general all messages should be either address the user with formal "Sie" or "Du" .. currently it seems to be mixed, e.g. see "Deine Antwort abschicken"

File: website/public/locales/de/dashboard.json [hunk 5]
```diff
@@ -1,8 +1,14 @@
...
+  "label": "Labeln",
+  "dashboard": "Dashboard",
   "go": "Los",
-  "grab_a_task": "Schnappen Sie sich eine Aufgabe!",
-  "label": "Label"
+  "welcome_message": {
+    "label": "Willkommen, {{username}}!",
+    "contributor": "Contributor",
+    "description": "Open Assistant ist ein open-source KI-Assistent welcher fortgeschrittene Language Models nutzt und trainiert, um Menschen zu verstehen und ihnen zu antworten.",
+    "instruction": "Schliesse Aufgaben ab um mit dem Training zu helfen und Punkte zu verdienen."
```
Comment (andreaskoepf) [inline website/public/locales/de/dashboard.json:12]: -> Schließe

File: website/public/locales/de/error.json [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  "err_1002": "Konte Aufgabe nicht nicht anerkennen",
+  "err_1003": "Ungültiger Antwortstyp",
+  "err_1004": "Interaktionsanfrage schlug fehl",
+  "err_1005": "Konnte keine Aufgabe generieren",
+  "err_1006": "Momentan gibt es keine Aufgaben vom Typ {{task_type}}.",
+  "err_1007": "Aufgabenverfügbarkeitsanfrage schlug fehl",
+  "err_1008": "Nachrichtenlänge zu groß",
+  "err_1009": "Ihre letzten Nachrichten enthalten Duplikate",
+  "err_1010": "Nachrichtentext ist leer",
+  "err_1011": "Sie können nicht zwei mal auf die selbe Nachricht antworten",
```
Comment (andreaskoepf) [inline website/public/locales/de/error.json:14]: -> "zweimal" & "dieselbe"

File: website/public/locales/de/index.json [hunk 7]
```diff
@@ -6,10 +6,20 @@
...
     "q1": "Wer organisiert die Entwicklung von Open Assistant?",
-    "a1": "Open Assistant ist ein Projekt, das von LAION und Einzelpersonen auf der ganzen Welt organisiert wird. Unser Ziel ist es das Open Assistant Sprachmodell allen Menschen frei zugänglich zu machen."
+    "a1": "Open Assistant ist ein Projekt, das von LAION und Einzelpersonen auf der ganzen Welt organisiert wird. Unser Ziel ist es das Open Assistant Sprachmodell allen Menschen frei zugänglich zu machen.",
+    "q2": "Welche Lizenz benutzt Open Assistant?",
+    "a2": "Der Code und die Models werden unter der Apache 2.0 Lizenz herausgegeben.",
+    "q3": "Werden die Trainingsdaten auch veröffentlicht?",
+    "a3": "Ja, unter CC BY 4.0.",
+    "q4": "Wird Open Assistant kostenlos sein?",
+    "a4": "Ja, Open Assistant wird man kostenlos benutzen und modifizieren können.",
+    "q5": "Welche hardware werde ich brauchen um die Models laufen zu lassen?",
```
Comment (andreaskoepf) [inline website/public/locales/de/index.json:16]: "Hardware" capitalized, maybe "Modelle" better than "Models" or even "Sprachmodelle"

File: website/public/locales/de/message.json [hunk 8]
```diff
@@ -1,20 +1,29 @@
 {
+  "confirm_open_link_body": "Möchten sie diesen Link öffnen?",
```
Comment (andreaskoepf) [inline website/public/locales/de/message.json:2]: "Sie" capitalized

File: website/public/locales/de/message.json [hunk 9]
```diff
@@ -1,20 +1,29 @@
...
   "parent": "Vorgänger",
   "reactions": "Reaktionen",
-  "recent_messages": "Kürzliche Nachrichten",
+  "recent_messages": "Kürzliche Nachrichten auf {{language}}",
   "report_action": "Melden",
   "report_placeholder": "Warum sollte diese Nachricht überprüft werden?",
   "report_title": "Meldung",
   "send_report": "Senden",
-  "stop_tree": "Stop tree",
+  "stop_tree": "Tree stoppen",
```
Comment (andreaskoepf) [inline website/public/locales/de/message.json:22]: Tree -> maybe "Baum" or "Nachrichtenbaum"?

**Review by andreaskoepf — CHANGES_REQUESTED** [review]
Really nice job! I left some comments about things that immediately caught my eye as native speaker.

File: website/public/locales/de/common.json (hunk 2 above)
Comment (Psychpsyo) [reply website/public/locales/de/common.json:7]: That button uses Title case. In English it's "Report a Bug", so I'd think German should also capitalize the verbs. (Or do we not have title case in German?)

File: website/public/locales/de/common.json (hunk 4 above)
Comment (Psychpsyo) [reply website/public/locales/de/common.json:41]: Ah, yea. I used the informal at first before noticing that the existing translations were using formal language so I missed this when switching it over.

**Comment by Psychpsyo** [comment]
> Really nice job! I left some comments about things that immediately caught my eye as native speaker.

Fixed all of those.

**Review by AbdBarho — APPROVED** [review]
Dankeschön!

**Review by andreaskoepf — APPROVED** [review]

**Review by Sindhuja365 — APPROVED** [review]

**Review by Sindhuja365 — APPROVED** [review]

**Pull request merged**